src/librustdoc/fold.rs

```diff
@@ -87,7 +87,7 @@ crate trait DocFolder: Sized {
     }
 
     fn fold_crate(&mut self, mut c: Crate) -> Crate {
-        c.module = c.module.take().and_then(|module| self.fold_item(module));
+        c.module = self.fold_item(c.module).unwrap();
```

Comment:
Is there ever a case where this `unwrap` will fail? Why does `fold_item` return an `Option` anyway?

Reply:
`fold_item` returns `None` if the itme was stripped. I don't think the whole crate can be stripped so this should be fine.